Python/dp/N-Queen.py

```diff
@@ -0,0 +1,91 @@
+class QueenChessBoard:
```

Comment:
Add description of the problem at the top.

Reply:
Done